plugins/SoundFontPlayer/SoundFontPlayer.cpp

```diff
@@ -780,7 +780,7 @@ void Sf2Instrument::noteOn( Sf2PluginData * n )
 }
 
 
-void Sf2Instrument::noteOff( Sf2PluginData * n )
+void SoundFontInstrument::noteOff( SoundFontPluginData * n )
```

Comment:
(Non-blocking) Per the Coding Conventions [re: parentheses](https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces), [re: pointers](https://github.com/LMMS/lmms/wiki/Coding-Conventions#pointers):

> 1. Spaces MUST NOT be added after an opening parenthesis or bracket
> 2. Spaces MUST NOT be added before a closing parenthesis or bracket

and,

> The `*` symbol in pointer declarations MUST be adjacent to the type, not the variable name. This rule also applies to `**`, `&`.

```suggestion
void SoundFontInstrument::noteOff(SoundFontPluginData* n)
```